README.md

```diff
@@ -17,6 +17,7 @@ a performance baseline of Kubernetes cluster on your provider.
 | [Smallfile](docs/smallfile.md) | Storage IO Performance | Working            |  Used, default : 3second  | Not Supported         |
 | [fs-drift](docs/fs-drift.md)   | Storage IO Longevity   | Working            |  Not used                  | Not Supported         |
 | [hammerdb](docs/hammerdb.md)   | Database Performance   | Working            |  Not used                  | Not Supported         |
+| [Service Mesh](docs/servicemesh.md) | Microservices     | Working            |  ?                         | Not Supported         |
```

Comment:
Looking through the code it seems that Reconciliation would be used, since you are maintaining state... so the default 3 second.